app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardActivityTest.kt

```diff
@@ -109,18 +109,16 @@ class RevisionCardActivityTest {
   private fun createRevisionCardActivityIntent(
     internalProfileId: Int,
     topicId: String,
-    subTopicId: Int
+    subtopicId: Int
   ): Intent {
     return RevisionCardActivity.createRevisionCardActivityIntent(
       ApplicationProvider.getApplicationContext(),
       internalProfileId,
       topicId,
-      subTopicId
+      subtopicId
     )
   }
 
-  // TODO(#59): Figure out a way to reuse modules instead of needing to re-declare them.
```

Comment:
This one should be kept; my earlier comment was only referring to the second of the TODOs.

Reply:
Sorry, @BenHenning misunderstood. Corrected it now.